common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/builders/IngestorConfigBuilder.scala

```diff
@@ -0,0 +1,41 @@
+package uk.ac.wellcome.pipeline_storage.builders
+
+import com.typesafe.config.{Config, ConfigException}
+import uk.ac.wellcome.pipeline_storage.models.IngestorConfig
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+object IngestorConfigBuilder {
+  def buildIngestorConfig(config: Config): IngestorConfig = {
+
+    // TODO: Work out how to get a Duration from a Typesafe flag.
+    val flushInterval = 1 minute
```

Comment:
Assuming for now this will be the same for all PS services?

Reply:
This will probably need to be modified per service, good spot